app/views/cars/_form.html.erb

```diff
@@ -1,3 +1,8 @@
+<%
+=begin%>
+  Add required feilds, so user does not reach the error screen
```

Comment:
Add required fields as in, make sure that the user can see that those fields are required and doesn't have an option to not fill it and unable to click on the submit unless the fields are filled?